README.md

```diff
@@ -1,20 +1,14 @@
 # conjur-api-python3
 
-Python3-based API SDK for [Conjur OSS](https://www.conjur.org/). The repo
-also includes a self-contained CLI tool (`conjur-cli`) that wraps the API
-in a simple executable script/binary.
+Command-line tool and programmatic Python access to the Conjur API to manage your Conjur resources. 
```

Comment:
Sigal I think we need to seperate the 2 offering this repo has:

1. Command-line tool to manage your Conjur resources. This CLI is self-contained.
2. Python SDK based on Conjur API, written in Python3

combining the 2 just makes it harder to read

Reply:
Is this in addition to the above sentence that yoou wat to add @sigalsax ??

From @InbalZilberman's comment:
- A command-line tool for managing your Conjur resources. This Conjur CLI is self-contained.
- Python SDK based on the Conjur REST API, written in Python3

Reply:
The follow-up sentence explains it a bit more

"This repository includes both the self-contained Conjur command-line tool (`conjur`) and Python3-based SDK for Conjur."

I prefer we leave it because both are used to manage Conjur resources. In the follow-up sentence we explain a bit more

Reply:
Sure I just think it is best to seperate the 2 as much as possible in the text and in visualization this is why i suggest: 1,2